backstop/tests/tree.html

```diff
@@ -0,0 +1,1613 @@
+                    aria-hidden="true"
+                  ></svg-placeholder>
+                  <span class="iui-node-label">
+                    <div class="iui-title"> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+                      tempor
+                      incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                      ullamco
+                      laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+                      voluptate
+                      velit
+                      esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt
+                      in
+                      culpa
+                      qui officia deserunt mollit anim id est laborum.
+                    </div>
+                  </span>
+                </div>
+              </div>
+            </li>
+          </ul>
+        </li>
+        <li>
+          <div
+            class="iui-tree-node"
+            tabindex="0"
+            aria-expanded="false"
+          >
+            <div class="iui-node-content">
+              <button
+                class="iui-button iui-borderless iui-small iui-expander"
+                type="button"
+              >
+                <svg-chevron-right
+                  class="iui-icon"
+                  aria-hidden="true"
+                ></svg-chevron-right>
+              </button>
+              <span class="iui-node-label">
+                <div class="iui-title"> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                  incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                  ullamco
+                  laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+                  velit
+                  esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+                  culpa
+                  qui officia deserunt mollit anim id est laborum.
+                </div>
+                <div class="iui-caption"> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                  incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                  ullamco
+                  laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+                  velit
+                  esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+                  culpa
+                  qui officia deserunt mollit anim id est laborum.
+                </div>
+              </span>
+            </div>
+          </div>
+        </li>
+      </ul>
+    </section>
+
+    <hr />
+    <h2>With Checkbox</h2>
```

Comment:
Shouldn't first "Subcategory" label align with the second one's label?
![image](https://user-images.githubusercontent.com/36186912/144001730-f482dc82-abd4-45eb-94f3-e4723cc14a56.png)

Reply:
Not sure, what do you think @FlyersPh9

Reply:
![image](https://user-images.githubusercontent.com/849817/144074231-a4758699-7247-4763-aeb6-cbedd290b1f5.png)
Good catch Vilius.  If a subcategory can not be expanded we should have empty space where the expand arrow would be.  Otherwise, if we had a mix of expandable and non-expandable subcategories the text would not align.

Reply:
Added left margin to `iui-node-label` when expander button is not present

Reply:
Text after icon is still misaligned:
![image](https://user-images.githubusercontent.com/36186912/144184913-a54f047c-4ea8-43ce-ac9a-0d577a203259.png)

Now I see that the icon is also misaligned with previous node text.

Reply:
Ok, it is weird situation, it seems we either align icon with expander or icon with label text.

Reply:
Fixed alignment